source/bios/damian_brady.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: author
+permalink: /bios/damian_brady/
+date: 2017-08-20 09:00
+authorId: damian_brady
+comments: false
+---
+
+Damian works for [Microsoft](https://microsoft.com/) as a Cloud Developer Advocate in the [League of Extraordinary
+Cloud DevOps Advocates](http://loecda.com) team.
+ 
+An ex-[Microsoft MVP](https://mvp.microsoft.com/en-us/mvp/Damian%20Brady-5000595) and developer at [Octopus Deploy](https://octopus.com), he's also a [Pluralsight Author](https://www.pluralsight.com/authors/damian-brady), and co-author of [Professional Team Foundation Server 2013](http://brdy.in/ProTFS2013) from Wiley. He's based in Toronto Canada, but when back in Australia, he co-organises the [Brisbane .NET User Group](http://www.meetup.com/Brisbane-Net-User-Group/), and the annual [DeveloperDeveloperDeveloper Brisbane](http://www.dddbrisbane.com/) conference.
+ 
+Damian regularly [speaks at conferences, User Groups, and other events](https://damianbrady.com.au/speaking) around the world, but most of the time you'll find him talking to developers and IT Pros to help them get the most out of their DevOps strategies.s
```

Comment:
Not just that file either - that's what I get for switching to Mac - `Ctrl-S` don't work the same.